### PR TITLE
Adds _char macros, bumps recommended version to 513.

### PR DESCRIPTION
**Pull request by MistakeNot4892**
Tested basic gameplay, savefiles, speech, paper, announcements, VV - everything seems stable other than paper and browser windows in general displaying mangled gibberish when Unicode is entered. I only did very limited and non-technical testing however.

### CODE REVIEW

**Comment by Lohikar** [comment]
It might be worth checking how functional https://github.com/Lohikar/Nebula/tree/513ification is before going with this PR, I forgot that Quard PR'd a bunch of additional 513 changes to it.

**Comment by MistakeNot4892** [comment]
I have pulled in the commits from that branch, I'm not sure what you mean by 'how functional'.

**Comment by Lohikar** [comment]
By 'don't know how functional' I mean 'I haven't tested this beyond checking it compiles'.

File: code/_513_char_macros.dm [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// These are placeholder defines to ensure sanitization etc works properly with Unicode
+// in DM 513, they should be replaced over time with the appropriate function call. 
+#if DM_VERSION >= 513
+#define length(X...)       length_char(X)
+#define copytext(X...)     copytext_char(X)
```
Comment (Lohikar) [inline code/_513_char_macros.dm]: I don't think this set of macros is needed with the other changes pulled in.

**Review by comma — APPROVED** [review]

**Comment by Lohikar** [comment]
I didn't get a chance to review this, I guess we're doing that live now.

**Pull request merged**